app/frontend/hooks/resources/use-jurisdiction.ts

```diff
@@ -18,10 +17,8 @@ export const useJurisdiction = () => {
     ;(async () => {
       try {
         setCurrentJurisdiction(jurisdictionId)
-        if (isUUID(jurisdictionId)) {
-          await fetchJurisdiction(jurisdictionId)
-          setCurrentJurisdiction(jurisdictionId)
-        }
+        await fetchJurisdiction(jurisdictionId)
+        setCurrentJurisdiction(jurisdictionId)
```

Comment:
Should there be some error handling if the jurisdiction does not exist?

Reply:
isnt this handled by the catch block?